packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -0,0 +1,555 @@
+                        )}
+                    </p>
+                    <p className='account-card__description-text--small'>{localize('Supported platform:')} </p>
+                    <p className='account-card__description-text--small'>
+                        {localize('DMT5')} <br />
+                    </p>
+                </AccountCard>,
+            ]}
+        />
+    );
+};
+
+class AccountTypesModal extends Component {
+    state = {
+        account_type_tab_index: 0,
+    };
+
+    setAccountTypeTabIndex = tab_index => {
+        this.setState({
+            account_type_tab_index: tab_index,
+        });
+    };
+
+    closeModal = () => {
+        this.props.toggleAccountTypesModal(false);
+    };
+
+    redirectToMt5 = account_type => {
+        this.closeModal();
+        this.props.history.push(`${routes.mt5}#${account_type}`);
+    };
+
+    redirectToMt5Real = () => {
+        if (!this.props.is_logged_in || this.props.is_mt5_allowed) {
+            this.redirectToMt5('real');
+            // TODO: [deriv-eu] Update this after EU account sign-up completion
+        } else {
+            window.open(urlFor('user/metatrader', undefined, undefined, true));
+        }
+    };
+
+    redirectToMt5Demo = () => {
+        this.redirectToMt5('demo');
+    };
+
+    // TODO: [deriv-eu] Update and use it after EU account sign-up completion
+    createRealAccount = () => {
+        if (this.props.can_upgrade_to === 'svg') {
+            this.props.openRealAccountSignup();
+        } else {
+            window.open(urlFor('new_account/maltainvestws', undefined, undefined, true));
+        }
+    };
+
+    render() {
+        return (
+            <Modal
+                title={localize('Account types')}
+                width='904px'
+                is_open={this.props.is_account_types_modal_visible}
+                toggleModal={this.closeModal}
+                has_close_icon={this.props.is_dismissible}
+            >
+                <ThemedScrollbars autoHide style={{ height: '63.5rem' }}>
+                    <div className='account-types'>
+                        <p className='account-types__intro'>
+                            <Localize
+                                i18n_default_text='{{ website_name }} offer various accounts based on 2 account types that suites different need. You can have all of them whenever you want. To view this page again, simply press any of the <0/> icon in Account Switcher.'
+                                values={{ website_name }}
+                                components={[
+                                    <Icon key={0} className='account-types__text-icon' icon='IcInfoOutline' />,
+                                ]}
+                            />
+                        </p>
+                        {boxGenerator({
```

Comment:
Please move the invocation of this function to line 493 (before the return statement of the `render()`). `ThemedScrollbars` renders at various intervals and triggers, we don't need to recalculate `boxGenerator` every time. Perhaps even a pure component if you're not performing any deep object operations?